### PR TITLE
calendar was created

### PR DESCRIPTION
**Pull request by BilokonYaroslav**
- [DEMO LINK](https://BilokonYaroslav.github.io/layout_calendar/)
- [TEST REPORT LINK](https://BilokonYaroslav.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--length_29">
+      <div class="calendar__day calendar__day--start-day_sun"></div>
+      <div class="calendar__day"></div>
```
Comment (IvanFesenko) [inline src/index.html]: `calendar__day` should be in `calendar` element

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: $calendar-padding;
+  width: $calendar-width;
+
```
Comment (IvanFesenko) [inline src/styles/blocks/calendar.scss]: To center calendar add this one
```suggestion
  width: $calendar-width;
  margin: 0 auto;
```

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  &__day {
+    @include square($square-size);
+    border: 1px black solid;
+    background-color: #eee;
+    font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+    font-size: 30px;
+    text-align: center;
+    line-height: $square-size;
+    transition: all 0.5s ease;
+
```
Comment (IvanFesenko) [inline src/styles/blocks/calendar.scss]: No need for transition all, use for transform and color only.
`transition: background-color 0.5s, transform 0.5s ease`

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job. Check my comments

File: src/index.html (hunk 1 above)
Comment (BilokonYaroslav) [reply src/index.html]: calendar__day in calendar block as it shuold.

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Not fixed. This how it should looks like
2023-02-16_15h37_19

**Review by YegorKorenkov — COMMENTED** [review]
Everything is okey. Nice work!

**Pull request closed without merging**